source/manual/howto-manually-remove-assets.html.md

```diff
@@ -4,35 +4,35 @@ title: Remove an asset
 section: Assets
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2018-08-09
+last_reviewed_on: 2018-08-15
 review_in: 6 months
 ---
 
 If you need to remove an asset manually from `assets.publishing.service.gov.uk`,
 follow these steps:
 
 1. `ssh backend-1.production`
-2. `govuk_app_console asset-manager`
-3. `asset = Asset.find("asset-id-from-url")` (e.g. `57a9c52b40f0b608a700000a`) or for a Whitehall asset `asset = WhitehallAsset.find_by(legacy_url_path: '/government/uploads/system/uploads/attachment_data/file/id/path.extension')`
-4. Make a note of the UUID for the asset: `asset.uuid` - this will be used to delete the file from Amazon S3
-5. Make a note of the path of the asset: `asset.file.path` - this will be used to delete the file from the file system
-6. Check the asset is what you think it is and mark it as deleted: `asset.destroy`
-7. Remove the asset from AWS S3
-    1. Log into AWS console and [assume a role][assume-role] for govuk-infrastructure-production
-    2. In the [govuk-assets-production S3 bucket][govuk-assets-production-bucket] search for a file named with the UUID identified in step 4
-    3. Delete this file
-8. Remove the asset from asset-manager file system (it may not be here as these are automatically removed after S3 upload)
+1. `govuk_app_console asset-manager`
+1. `asset = Asset.find("asset-id-from-url")` (e.g. `57a9c52b40f0b608a700000a`) or for a Whitehall asset `asset = WhitehallAsset.find_by(legacy_url_path: '/government/uploads/system/uploads/attachment_data/file/id/path.extension')`
+1. Make a note of the path of the asset: `asset.file.path` - this will be used to delete the file from the file system
+1. Mark the asset as deleted (and remove it from S3 if necessary - this will prevent restoration)
+   - [`rake assets:delete[<asset.id>]`][rake-delete]
+   - [`rake assets:delete_and_remove_from_s3[<asset.id>]`][rake-delete-and-remove-from-s3]
+1. Remove the asset from asset-manager file system (it may not be here as these are automatically removed after S3 upload)
```

Comment:
Is this really necessary?

Reply:
I reckon probably not, but it was here before I started, and there is a tiny chance it's still there.